.github/workflows/comment-on-asciidoc-changes.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Comment on PR for .asciidoc changes
+
+on:
+  pull_request:
+    types:
+      - synchronize
+      - opened
+      - reopened
+    branches:
+      - main
+      - master
+      - "9.0"
```

Comment:
Branches `master` and `9.0` do not exist.
```suggestion
```

Reply:
Oh. I was made aware of that `9.0` will be there in the next days due to FF? is this correct?

Then we should re-add 9.0.

Reply:
> Oh. I was made aware of that 9.0 will be there in the next days due to FF? is this correct?

Indeed. Standard branch policies will add the 9.0 branch at FF, which officially is today.